File: src/Lession5/Ex2.java
package Lession5;

public class Ex2 {
    static void main() {
        int[] array = new int[] {-12, 7, -3, 15, -9, 4, -17, 2, 19, -6, 11, -14, 0, -5, -2, 16, -5, 13, -1, 10};

        int num_even = 0;
        int num_odd = 0;

        for (int k = 0; k < array.length; k++) {
            if (array[k] % 2 == 0) {
                num_even++;
            } else {
                num_odd++;
            }
        }

        System.out.println("Count of even numbers: " + num_even);
        System.out.println("Count of odd numbers: " + num_odd);
    }
}

File: src/Lession5/Ex2.kt
package Lession5

fun main() {
    val array = intArrayOf(-7, 14, -3, 18, -12, 5, -18, 2, 11, -8, 0, 17, -4, 9, -15, 6, 0, 13, -10, 4)

    var num_even = 0;
    var num_odd = 0;

    for (i in array) {
        if (i % 2 == 0) {
            num_even++
        } else {
            num_odd++
        }
    }

    println("Count of even numbers: $num_even")
    println("Count of odd numbers: $num_odd")
}
